app/views/citizens/other_assets/_second_home_conditional_checkbox.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-checkboxes__item">
+  <input class="govuk-checkboxes__input"
+         id="second_home"
+         name="second_home_checkbox"
+         type="checkbox"
+         value="yes"
+         <%= @form.second_home_checkbox_status %>
```

Comment:
As mentioned above - I think `@form.checkbox_status(:second_home)` would be a better pattern.